### PR TITLE
removed needless const declarations

### PR DESCRIPTION
**Pull request by jatin69**
For now, I've just updated the file with most `const` declarations. Kindly verify the changes. Address Issue #44

### CODE REVIEW

**Comment by jatin69** [comment]
@RubyNova I've made all the necessary changes. I'd suggest to review it very carefully, so you don't have to worry about const redundancies later.

File: src/NovelImageRect.cpp [hunk 1]
```diff
@@ -10,9 +10,9 @@ namespace NovelRT {
 NovelImageRect::NovelImageRect(NovelLayeringService* layeringService,
                                const float& screenScale,
                                const GeoVector<float>& size,
-                               const std::string_view imageDir,
-                               const NovelCommonArgs& args,
-                               const GLuint programId,
+                               std::string_view imageDir,
+                               NovelCommonArgs& args,
```
Comment (RubyNova) [inline src/NovelImageRect.cpp]: This is passed by reference. Please re-add the `const` qualifier here.

Comment (jatin69) [reply src/NovelImageRect.cpp]: oh, my bad. I'll re-add it.

**Comment by jatin69** [comment]
@RubyNova Let me know if you want me to rebase it, or else you could just squash it.

**Comment by RubyNova** [comment]
I'll just squash it, CLAssistant is being slow today, I don't feel like waiting for it twice, haha.

**Pull request merged**